database-commons/src/main/java/io/cdap/plugin/db/sink/AbstractDBSink.java

```diff
@@ -114,6 +127,45 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
       validateSchema(collector, driverClass, dbSinkConfig.getTableName(), inputSchema, dbSinkConfig.getDBSchemaName());
     }
   }
+  public void validateOperations(FailureCollector collector, T dbSinkConfig, @Nullable Schema inputSchema) {
+    if (dbSinkConfig.containsMacro(OPERATION_NAME) || dbSinkConfig.containsMacro(RELATION_TABLE_KEY)) {
+      return;
+    }
+    String operation = dbSinkConfig.getOperationName();
+    String relationTableKey = dbSinkConfig.getRelationTableKey();
+    if (operation.equalsIgnoreCase(Operation.INSERT.getName()) && relationTableKey == null) {
```

Comment:
this condition check is not required, if user puts some wrong value also, we will not consider it

Reply:
done

Reply:
this condition is required as for INSERT operation we don't need to go further and check for field values.